examples/public-repository-complete-example/main.tf

```diff
@@ -37,23 +76,23 @@ module "repository" {
       enforce_admins         = true
       require_signed_commits = true
 
-      required_status_checks = {
+      required_status_checks = [{
         strict   = true
         contexts = ["ci/travis"]
-      }
+      }]
 
-      required_pull_request_reviews = {
+      required_pull_request_reviews = [{
         dismiss_stale_reviews           = true
         dismissal_users                 = ["terraform-test-user-1"]
         dismissal_teams                 = [github_team.team.slug]
         require_code_owner_reviews      = true
         required_approving_review_count = 1
-      }
+      }]
 
-      restrictions = {
-        users = ["terraform-test-user-1"]
+      restrictions = [{
+        users = ["mariux"]
```

Comment:
don't use your own user here

Reply:
We do not have a permanent test user as of now.. so it will fail the test.. must be an accepted member..

Reply:
ah yeah that makes sense. Shall I create a new test user and add it to the organization permanently?

Reply:
let's get it merge and improve later?